manual/manual_compiler.rst

```diff
@@ -137,6 +114,25 @@ This set of rebases are provided for convenience, but the facility is available
 
     print(circ.get_commands())
 
+For some gatesets, it is not even necessary to specify the CX and TK1 decompositions: there is a useful method :py:meth:`auto_rebase_pass` which can take care of this for you. The pass returned is constructed from the gateset alone. An example is given in the "Combinators" section below.
```

Comment:
```suggestion
For some gatesets, it is not even necessary to specify the CX and TK1 decompositions: there is a useful function :py:meth:`auto_rebase_pass` which can take care of this for you. The pass returned is constructed from the gateset alone. An example is given in the "Combinators" section below.
```

Reply:
worth noting here also that it relies on some pre-known decompositions, and may throw an error if no suitable known decompositions exist.